src/aind_data_schema_models/organizations.py

```diff
@@ -1125,3 +1125,12 @@ def from_name(cls, name: str):
         Organization.OTHER,
     ]
 )
+
+Organization.CATHETER_IMPLANTS = one_of_instance(
```

Comment:
maybe should be `CATHETER_IMPLANT_ORGANIZATIONS`?

Reply:
also I think this is supposed to go in the template, not organizations.py, which is generated